src/build.py

```diff
@@ -21,6 +21,7 @@
 import os
 import shutil
 import sys
+import textwrap
```

Comment:
I suppose since JF insisted we defer/avoid import of argparse into `ParseArgs()` that we ought to do the same with textwrap.